openwisp_ipam/tests/test_admin.py

```diff
@@ -339,3 +340,32 @@ def test_change_view_master_multitenant(self):
         url = reverse(f'admin:{self.app_label}_subnet_change', args=[child1.pk])
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+
+    def test_admin_group_permission(self):
+        admin = Group.objects.get(name="Administrator")
+        admin_permissions = [
+            'add_ipaddress',
+            'change_ipaddress',
+            'delete_ipaddress',
+            'view_ipaddress',
+            'add_subnet',
+            'change_subnet',
+            'delete_subnet',
+            'view_subnet',
+        ]
+        perms = list(admin.permissions.values_list('codename', flat=True))
+        for p in admin_permissions:
+            self.assertIn(p, perms)
+
+    def test_operator_group_permission(self):
```

Comment:
Is it possible to write another test to ensure that operator does not have permissions that it's not suppose to have?

Reply:
Okay @atb00ker, I will try to add that too if possible.

Reply:
Great. Other than this, Looks good from my end. Tested. it's working.
Feel free to take another issue when you are done with updating this and waiting for Fed to review / merge it! :smile: